tech-project-back-end/DTO/CategoryDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace tech_project_back_end.DTO
+{
+    public class CategoryDTO
+    {
+        public string CategoryId { get; set; }
+        public string CategoryName { get; set; }
+        public bool IsDeleted { get; set; } = false;
```

Comment:
Read Supplier, SupplierDTO. Supplier extends BaseEntity class. It will extend many attribute like is_deleted, created_date.
why you use IsDeteted in DTO. The attibute of base entitiy class must dont be sent to client